packages/cli/lib/DevServerManager.js

```diff
@@ -61,6 +70,7 @@ class DevServerManager {
     this.debug = debug;
     this.componentsByType = this.arrangeComponentsByType(components);
 
+    await startPortManagerServer();
```

Comment:
My comment [here](https://github.com/HubSpot/hubspot-local-dev-lib/pull/42/files#r1361231085) is relevant. It should enable you to catch port conflicts when attempting to start up the server manager and log a useful error here